common/element_handle.go

```diff
@@ -63,7 +63,13 @@ func (h *ElementHandle) boundingBox() (*Rect, error) {
 func (h *ElementHandle) checkHitTargetAt(apiCtx context.Context, point Position) (bool, error) {
 	frame := h.ownerFrame(apiCtx)
 	if frame != nil && frame.parentFrame != nil {
-		element := h.frame.FrameElement().(*ElementHandle) //nolint:forcetypeassert
+		var (
+			el          = h.frame.FrameElement()
+			element, ok = el.(*ElementHandle)
+		)
+		if !ok {
+			return false, fmt.Errorf("unexpected type %T", el)
```

Comment:
Should we `k6ext.Panic()` in these cases? It would be consistent with the `FrameSession.onTargetCrashed` change, and it seems like something we should panic on.

Or will this panic higher up, as @ankur22 mentioned?

Reply:
Yes, it should be safe. It will eventually panic in the higher layers. For example, [`checkHitTargetAt`](https://github.com/grafana/xk6-browser/blob/fc34711f5ea6e1178bb10a059e77aa544eddccfc/common/element_handle.go#L63) returns err => [`newPointerAction`](https://github.com/grafana/xk6-browser/blob/fc34711f5ea6e1178bb10a059e77aa544eddccfc/common/element_handle.go#L1435) returns err => [`Click`](https://github.com/grafana/xk6-browser/blob/fc34711f5ea6e1178bb10a059e77aa544eddccfc/common/element_handle.go#L745) panics on the error.

Reply:
OK. Still, I think we should panic as close as possible to where the issue happens, so here, but it's fine, we can leave it as is.